testsuite/tests/lib-random/parallel.ml

```diff
@@ -0,0 +1,27 @@
+(* TEST *)
+let () = Random.init 42
+
+let f () =
+  let a = Random.int 100 in
+  let b = Random.int 100 in
+  let c = Random.int 100 in
+  (a, b, c)
+
+(* We intentionally spawn many more domains than hardware threads, to
+   ensure that we see scheduling effects that make the computations
+   are scheduled in a non-deterministic order. With a small number of
```

Comment:
The `are` is probably a typo?
Maybe just
```
that we see computations scheduled in a a non-deterministic order.
```
?

Reply:
Thanks, fixed.